e2e/browserslist/package-json/index.mjs

```diff
@@ -2,7 +2,7 @@ import assert from 'assert';
 import plugin from 'postcss-preset-env';
 import postcss from 'postcss';
 
-const result = await postcss([plugin()]).process(':any-link { color: blue; }').css;
+const result = (await postcss([plugin()]).process(':any-link { color: blue; }')).css;
```

Comment:
unrelated, but I noticed that this test never ran and was broken when adding the extra test for all these plugins together